docs/tutorials/how-to-override-section-components.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "How to override or customize native section components"
+sidebar_label: "How to override or customize native section components"
+---
+
+
+# How to override or customize native section components
+This guide covers how to override native section components or pass additional props to them.
+
+Native sections are provided by default by VTEX's `@faststore/core` package. Sections are composed by different components that can be customized or overriden to add functionalities or to change looks.
+
+{Image or gif of the expected behavior after completing the guide}
+
+---
+
+## Before you start
+
+_Before you begin, you should already have:_
+- A functioning Evergreen store.
+
+_You should also consider that:_
+
+- This feature is experimental.
```

Comment:
This won't be experimental when we launch, will it?

Reply:
I think it's ok to keep it like that. It's safer to say that it is an experimental feature 👍 hihi

Reply:
Since it's experimental, does it require any special config. or any permission? Can we list any behavior the dev could expect working with this feature?